### PR TITLE
Change list of screen types

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
- Add table create_types
- Add model, factory and seeder
- Modify the type of list when creating the screen.

### CODE REVIEW

File: ProcessMaker/Http/Controllers/Process/ScreenController.php [hunk 1]
```diff
@@ -18,15 +19,12 @@ class ScreenController extends Controller
...
     {
-        $types = [
-            '' => __('Select Type'),
-            'DISPLAY' => 'Display',
-            'FORM' => 'Form',
-        ];
-
-        if (Script::where('key', 'processmaker-communication-email-send')->exists()) {
-            $types['EMAIL'] = 'Email';
+        $screenTypes = ScreenType::all();
```
Comment (velkymx) [inline ProcessMaker/Http/Controllers/Process/ScreenController.php]: Use pluck instead https://laravel.com/docs/5.8/collections#method-pluck

Comment (marcoAntonioNina) [reply ProcessMaker/Http/Controllers/Process/ScreenController.php]: It is changed to use  method pluck

File: ProcessMaker/Http/Controllers/Process/ScreenController.php [hunk 2]
```diff
@@ -18,15 +19,9 @@ class ScreenController extends Controller
      */
     public function index()
     {
-        $types = [
-            '' => __('Select Type'),
-            'DISPLAY' => 'Display',
-            'FORM' => 'Form',
-        ];
+        $screenTypes = ScreenType::pluck('name', 'name');
```
Comment (velkymx) [inline ProcessMaker/Http/Controllers/Process/ScreenController.php]: merge these two statements pluck()->all();

Comment (marcoAntonioNina) [reply ProcessMaker/Http/Controllers/Process/ScreenController.php]: fixed up

**Pull request merged**